src/include/containers/IntrusiveQueue.hpp

```diff
@@ -95,6 +95,32 @@ class IntrusiveQueue
 		return ret;
 	}
 
+	bool remove(T removeNode)
```

Comment:
Is there a test for this one?

Reply:
Working on it. https://github.com/PX4/Firmware/pull/12246